luthier/src/main/java/com/yahoo/bard/webservice/application/LuthierBinderFactory.java

```diff
@@ -54,6 +56,14 @@ protected ConfigurationLoader getConfigurationLoader() {
         return Optional.empty();
     }
 
+    /**
+     * Temp stuff.
```

Comment:
We should flesh this out while we're here.

Reply:
This is marked as Temp stuff not because that it is incorrect (in fact it is exactly how we should implement in the future), but because the `concept` implementation on @michael-mclawhorn's branch will replace this with a wrapper. 

this will get changed whenever that PR is ready.